### DISCUSSION TITLE
v0.3.1-alpha

### DISCUSSION THREAD

**Discussion by aecelaya** [mist-medical/MIST#18] [Announcements]
Fix pyproject - Bug was not letting scripts be called after pip install

Add boundary loss functions

Add support for transfer learning with previous MIST models

<hr /><em>This discussion was created from the release <a href='https://github.com/aecelaya/MIST/releases/tag/v0.3.1-alpha'>v0.3.1-alpha</a>.</em>

**Comment by aecelaya** [comment]
A very subtle yet insidious bug in the latest versions of MIST allowed you to install it via PyPI (i.e., pip install) but didn't let you see the scripts after that. I never noticed it since it didn't appear to be an issue in my test environment. However, the bug should be fixed now. Sorry if this caused any confusion.

In addition to the bug fix, I added support for boundary loss functions and transfer learning via pretrained MIST models. I'm hoping to put models trained on public data on git lfs (and possibly hugging face) soon.

As always, please let me know if you have any questions and let me know about any bugs via email, by opening an issue, or by a pull request on the github repo.
